### ISSUE TITLE
Optimize Ways to claim ownership

### ISSUE THREAD

**Issue by aditya-manit** [graphprotocol/mission-control-curator#226]
Right now the ways to claim ownership of a project are really not optimal and totally relies on the person to view the tweet for the project that you are requesting. There might be a case when he would never see that tweet. Basically we want to avoid this.

I suggest the following two ways:

1) **Easy Way**: When a person X adds a project, give the person an option to add his social handles (twitter for example), and when someone (say Y) wants to request ownership, he will tweet and mention the user X. This increases the chances of user X seeing the request from user Y.

2) **Complex Way:**  When user X adds a project, he/she should be given an option to claim the project. A way to do that he has tweet from the official twitter handle of the project about claiming the project. If he tries to fake it, someone might challenge the project later. 
And if he is not an official member of the project is should be given an option to set an price for user Y (who wants to claim the project), if user Y pays that amount ownership will be transferred to Y. (This can be done using an smart contract)

Drawback: X might charge a really high amount --> To solve this, we can put limit to amount an user can charge for an new project. We can be governed by some curve, as people buy or sell that project price will increase.

**Comment by koen84** [comment]
Buying project ownership i hope to never see, it's not in the spirit and will open the door for abuse.

**Comment by yanivtal** [comment]
This flow is for sure suboptimal right now. I think the solution to this will be Web3 messaging where the listing owner can be notified and easily take action.

**Comment by aditya-manit** [comment]
Yup !! we may integrate with Status for this maybe (https://status.im/ - Status is a secure messaging app, crypto wallet, and Web3 browser.)
